tests/unit/dataloader/test_tf_dataloader.py

```diff
@@ -20,18 +20,22 @@
 import time
 import timeit
 
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.metrics import roc_auc_score
```

Comment:
thanks for fixing the import order here!

I do wonder how the old version got through the existing isort check though -

Reply:
It looks like this might be an edge case in isort. It doesn't seem to spot any imports in the wrong order after the try/except.